apps/api/v1/lib/validations/booking.ts

```diff
@@ -23,6 +23,7 @@ export const schemaBookingGetParams = z.object({
   dateTo: iso8601.optional(),
   order: z.enum(["asc", "desc"]).default("asc"),
   sortBy: z.enum(["createdAt", "updatedAt"]).optional(),
+  upcoming: z.enum(["true", "false"]).optional(),
```

Comment:
I think it's better to have type Status = "upcoming" to which we later on can add other statuses like type Status = "upcoming" | "past" and then use it as status: Status.

Rationale: it is very likely we will want to filter also by past, cancelled, recurring etc. so we would have to add many properties like pastOnly, and instead we can replace that logic with 1 property status: Status.

Reply:
Ideally this would be an array `status: Status[]` that would allow passing multiple statuses, but for now `status: Status` should be good enough because in the future we will migrate to another api version (this is version 1 and we are working on version 2)